### ISSUE TITLE
Support for manganelo/mangakakalot

### ISSUE THREAD

**Issue by DigitalCorgi** [anka-213/webcomic_reader#139]
I would really like to see support added for this site, or I would like some help trying to figure out how to add it myself, as it said in the readme to ask for help.

**Comment by pandavova** [comment]
yeah would love support for mangakakalot aswell, it should be possible because every img is on the page, i just dont know how to do it.
